src/test/java/OpenPageTest.java

```diff
@@ -1,10 +1,12 @@
+import models.CustomerModel;
```

Comment:
It seems like unncecessary import

Reply:
Useful IntelliJ shortcuts:
- Optimize Imports: ⌃⌥O (macOS) / Ctrl+Alt+O (Windows/Linux)
- Reformat File: ⌘⌥L (macOS) / Ctrl+Alt+L (Windows/Linux)

Reply:
Thanks guys! Yes, it isn't necessary import.